Data/HashMap/Strict.hs

```diff
@@ -249,6 +254,51 @@ alter f k m =
     Just v  -> insert k v m
 {-# INLINABLE alter #-}
 
+-- | /O(log n)/  The expression (@'alterF' f k map@) alters the value @x@ at
+-- @k@, or absence thereof. @alterF@ can be used to insert, delete, or update
+-- a value in a map.
+--
+-- @since 0.2.9
```

Comment:
Docs should mention that `alterF` is a flipped version of `Control.Lens.At.at`.

Reply:
Done (in Base and Strict modules).